content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -1,59 +1,66 @@
 ---
 linktitle: Creating Projects Using the CLI
 title: Creating Projects Using the Command Line Interface
-description: Learn how to create and build new Open 3D Engine (O3DE) projects from the default project template using the CLI.
+description: Learn how to create and build new Open 3D Engine (O3DE) projects from the default project template using the command line (CLI).
 weight: 200
 toc: true
 ---
 
-The tutorial on this page and the video below provide an introduction to project configuration and building in Open 3D Engine (O3DE) using the command line interface (CLI).
+The tutorial on this page and the video below provide an introduction to project configuration and building in Open 3D Engine (O3DE) using a command line interface (CLI).
 
 The instructions here and in the video guide you through the following steps:
 
 * Create a new O3DE project using the CLI.
 * Create Visual Studio project files in your O3DE project directory.
 * Build your O3DE project.
 
-At the end of the tutorial you'll have a new O3DE project based on the default project template, opened in the **O3DE Editor**.
+At the end of the tutorial you'll have a new O3DE project, based on the default "standard" project template, opened in **O3DE Editor**.
```

Comment:
What's the reason to add `standard` in here? Original wording seems fine, since it is created from "the default project template". If the default template can be swapped, maybe the "base game template" or something more specific? There will also be different "standard" project types for different O3DE consumers.

Reply:
"standard" is the actual name of the default template now.